specs/networking/libp2p-standardization.md

```diff
@@ -39,9 +59,10 @@ Current defaults are:
 - Digests: `SHA256` (also supports `SHA512`)
 
 *Note: Secio is being deprecated in favour of [TLS
-1.3](https://github.com/libp2p/specs/blob/master/tls/tls.md). It is our
-intention to transition to use TLS 1.3 for encryption between nodes, rather
-than Secio.*
+1.3](https://github.com/libp2p/specs/blob/master/tls/tls.md) or
+[Noise](https://perlin-network.github.io/noise/). It is our intention to
```

Comment:
pretty sure we are talking about this Noise: https://noiseprotocol.org/noise.html

not the linked noise

Reply:
Oh yep. Nice catch. Will update. Thanks